com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -293,6 +322,10 @@ public void BeginFrame(CommandBuffer cmd, HDCamera camera, HDRenderPipeline hdIn
             // Override full screen anti-aliasing when doing path tracing (which is naturally anti-aliased already)
             m_AntialiasingFS        &= !m_PathTracing.enable.value;
 
+            // Antialiasing of any sort is disabled on DRS Pre post processes.
+            // The assumption here is that upsamplers that work before post processes should perform anti aliasing as well in one go.
```

Comment:
Uhm this might be problematic without first changing the UI a bit, this will disregard what the user selects in the options and is not great UX.

Reply:
Yes indeed, there is a bit of things we have to do in terms of UI.
For example, we still want the user to specify a fallback config, because existance of <name of secret upscaler> might not be guaranteed. I will do another pass on UI and add the dummy upscaler as a selection, we can do a UI pass.

Reply:
@FrancescoC-unity I have done some UI settings. First of all, I am calling this upscaler 'Prepost Upscaler' but later we will replace it with the proper name, since legally I cant mention it here ;) so assume that.
The UI Without enabling DRS for the camera now looks like this 
![image](https://user-images.githubusercontent.com/789671/108254734-66f71a00-7129-11eb-8028-b08c23cc7d91.png)

1) User selects DRS checkbox on camera
2) User sees the new technique as a checkbox. User presses it
3) Upon pressing it, the Post Anti-Aliasing selected gets indented, and changes its text to "Fallback Post Anti-Aliasing"

The reason for this fallback is because this upscaler could not be supported by the user's hardware. So the option must exist along a fallback.

I added the possibility of "Detected" in the text "Using PrePostProcess Upscaler" text, which would indicate the hardware running this would support it.

Let me know your thoughts

Reply:
Sounds good to me as a concept indeed, but given how the settings on the upscaling and dynamic resolution parameters are on the Asset (we don't allow per-camera settings for now), if we can detect what filtering is selected (and we should be able to?), we can skip the checkbox altogether and maybe just display info box + the fallback nomenclature.

Reply:
Updated this. Will continue conversaton wtih Maddalena